tests/test_runner.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import warnings
+from memo import memlist, Runner, grid
+
+
+@pytest.mark.parametrize("kw", [{"backend": "loky", "n_jobs": -1}, {"backend": "threading", "n_jobs": -1}, {"backend": "multiprocessing", "n_jobs": -1}])
+def test_base_multiple_calls(kw):
+    data = []
+    g = [{"a": 1}] * 100
+
+    @memlist(data=data)
+    def count_values(**kwargs):
+        return {"sum": sum(kwargs.values())}
+
+    runner = Runner(**kw)
+    runner.run(func=count_values, settings=g, progbar=True)
+    assert len(data) == len(g)
+
+
+@pytest.mark.parametrize("kw", [{"backend": "loky", "n_jobs": -1}, {"backend": "threading", "n_jobs": -1}, {"backend": "multiprocessing", "n_jobs": -1}])
+def test_keys_included(kw):
+    data = []
+    g = [{"a": 3, "b": 4, "c": 5}]
+
+    @memlist(data=data)
+    def count_values(**kwargs):
+        return {"sum": sum(kwargs.values())}
+
+    runner = Runner(**kw)
+    runner.run(func=count_values, settings=g, progbar=True)
+    assert len(data) == 1
+    assert all([k in data[0].keys() for k in g[0].keys()])
+
+
+@pytest.mark.parametrize("kw", [{"backend": "loky", "n_jobs": -1}, {"backend": "threading", "n_jobs": -1}, {"backend": "multiprocessing", "n_jobs": -1}])
```

Comment:
Since `n_jobs` is constant in all these examples, do we really need it to be parametrized?